src/fparser/Fortran2003.py

```diff
@@ -6063,7 +6103,27 @@ def match(string):
                 i2 = i2.lstrip()
                 return c, W(i1), M(i2), None
             return c,W(line), None, None
-        if c in ['E','F','G']:
+        if c in ['E']:
```

Comment:
The change is necessary because (apparently) Fortran supports ES and EF format specifiers as well as the plain E. Who knew? (Apparently the NEMO developers..)